src/main/webapp/app/exercises/programming/shared/utils/programming-exercise.utils.ts

```diff
@@ -90,3 +91,17 @@ export const hasDeadlinePassed = (exercise: ProgrammingExercise) => {
     }
     return referenceDate.isBefore(moment());
 };
+
+/**
+ * Removes the login from the repositoryURL
+ */
+export const createAdjustedRepositoryUrl = (participation: ProgrammingExerciseStudentParticipation) => {
+    let adjustedRepositoryURL = participation.repositoryUrl || '';
+    if (participation.student && participation.repositoryUrl) {
+        const userName = participation.student.login + '@';
```

Comment:
can we also consider the case for teams here?

Reply:
I have checked that and teams don't have a  preceding `username@` in the url